agent/structs/structs.go

```diff
@@ -72,6 +72,43 @@ const (
 	SystemMetadataRequestType                   = 31
 )
 
+// requestTypeStrings is used for snapshot enhance
+// any new request types added must be placed here
+var requestTypeStrings = map[MessageType]string{
+	0:  "Register",
+	1:  "Deregister",
+	2:  "KVS",
+	3:  "Session",
+	4:  "ACL", // DEPRECATED (ACL-Legacy-Compat)
+	5:  "Tombstone",
+	6:  "CoordinateBatchUpdate",
+	7:  "PreparedQueryRequest",
+	8:  "TxnRequest",
+	9:  "AutopilotRequest",
```

Comment:
Since the nouns above don't have request in them, maybe these shouldn't either:
```suggestion
	7:  "PreparedQuery",
	8:  "Txn",
	9:  "Autopilot",
```